framework/src/outputs/Console.C

```diff
@@ -366,7 +366,7 @@ Console::output(const ExecFlagType & type)
       _old_nonlinear_norm = std::numeric_limits<Real>::max();
 
     _console << std::setw(2) << _nonlinear_iter
-             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << '\n';
+             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << std::endl;
```

Comment:
Not losing the output. I guess I can remove the flush here, because we must have a flush already in iterations somewhere. The flush in action is more important to me. Also we are not flushing files but just screen. The impact should be very small.